srcpkgs/badwolf/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'badwolf'
 pkgname=badwolf
 version=1.2.2
-revision=1
+revision=2
 build_style=configure
 make_check_target=test
 hostmakedepends="pkg-config gettext"
-makedepends="webkit2gtk-devel libsoup-devel"
+makedepends="libwebkit2gtk41-devel"
```

Comment:
I think it's better to depend on `libsoup3-devel` explicitly here.

Reply:
hmm, there does indeed seem to be more packages that define both `webkit2gtk-devel` and `libsoup3-devel` explicitly than only the former (quick two-pass grep shows 9/16 not including badwolf). guess it wouldn't hurt

Reply:
I just think it's better to have explicit, instead of implicit, dependencies. :)